client/analysis/output.go

```diff
@@ -164,7 +164,7 @@ func prepareAllSummary(analysis types.Analysis) {
 	outputJSON.Summary.GitleaksSummary.LowVuln = len(outputJSON.GenericResults.HuskyCIGitleaksOutput.LowVulns)
 	outputJSON.Summary.GitleaksSummary.MediumVuln = len(outputJSON.GenericResults.HuskyCIGitleaksOutput.MediumVulns)
 	outputJSON.Summary.GitleaksSummary.HighVuln = len(outputJSON.GenericResults.HuskyCIGitleaksOutput.HighVulns)
```

Comment:
I believe you should update `GitleaksSummary.NosecVuln` with its length here. This way it should output the correct value on line 183.

Reply:
Great catch! Thanks a lot for 6023766.

Reply:
Fixed it in [6023766](https://github.com/globocom/huskyCI/pull/450/commits/6023766763a83f8b2286c7f71b2cc5e10b95a775)